plan.md

```diff
@@ -1,9 +1,282 @@
+communications so that the *whole team* can see and contribute to these.
+
+Our CRM *code* will be **fully open source**, following our [guiding principles](https://github.com/dwyl/phase-two#guiding-principles) and
+removing as many barriers to entry/use as possible for small businesses with a
+social impact. The data however, will not be accessible to anyone outside of the
+organisation or sold to 3rd parties (a la [LinkedIn](https://www.forbes.com/sites/georgeanders/2012/06/27/how-linkedin-strategy/#c4e2c1fdf82c)).
+
+Value lies in:
++ Linking all of the branches of our business together
++ Excellent low risk learning project for learning our new stack
++ Modular architecture which is easy to extend *without* becoming super patchy
+  + Modules re-used across multiple products and services projects
++ Transparency on business development for all of our team
++ Central location where data can easily be entered as our business development
+efforts expand with [Services](#services) and [Training](#training) offerings
++ 50% of all charitable organisations we have spoken to have told us that their
+current CRM is not fit for purpose, we could easily begin to provide a solution
+to this problem if we desired
+
+> There are many 'small' enhancements we will be making to increase the effectiveness
+of our own workflow. **Each should be evaluated as a potential product.**
+
+
+### Services
+
+Our Services arm is how we help other organisations serve _their_ communities and achieve their social goals.
+
+We *already* do far more than just 'web development' for our services clients
+and this should be formalised into our offering.
+
+#### ATM Workshops  
+Deeply understanding the problem and helping organisations to arrive at a
+solution with actionable metrics is where dwyl adds the most value.
+
+Our 1 to 5 day workshops will be the cornerstone of the Services offering,
+providing clients with:
++ Understanding the "success drivers" i.e. "ATMs"
++ Proposed technical path forward - what technology can the client use to be more effective at reaching their organisational goals
++ Simple prototype
++ Effective [agile collaboration and user testing training](#training)
+
+These workshops will be our largest 'per man hour' contributor to fixed overheads
+from our Services branch by the end of 2017.
+
+#### Product Development
+_Everything_ we build must have a social impact or reusable open source components.
+
+Our [processes](https://github.com/dwyl/contributing) for the delivery of product
+development and [working with services clients](https://github.com/dwyl/ateam-master-reference) are already highly
+developed. These will remain in flux and be [continuously improved]((https://github.com/dwyl/dwylbot)
+as we build our own [products](#product).
+
+#### Focus
++ Delighting our existing customers
++ Writing out the structure for our ATM workshops
++ Building sales through delivering great work and sharing "success stories" (case studies)
```

Comment:
Having thought more through this, it's mostly about building this into a super solid foundation that can function independently of use. All of these are components 👍